doc/rst/meta/modules/standard.rst

```diff
@@ -6,19 +6,111 @@ Standard Modules
 Standard modules are those which describe features that are considered
 part of the Chapel Standard Library.
 
+
+Automatic Modules
+-----------------
+
 All Chapel programs automatically ``use`` the modules :chpl:mod:`Builtins` ,
 :chpl:mod:`Math` , and :chpl:mod:`Types` by default.
 
+.. toctree::
+   :maxdepth: 1
+
+   Builtins <standard/Builtins>
+   Math <standard/Math>
+   Types <standard/Types>
+
+
+Data Structures
+---------------
+
+.. toctree::
+   :maxdepth: 1
+
+   Heap <standard/Heap>
+   List <standard/List>
+   Map <standard/Map>
+   Set <standard/Set>
+
+   
+Diagnostics
+-----------
+
+.. toctree::
+   :maxdepth: 1
+
+   CommDiagnostics <standard/CommDiagnostics>
+   Memory <standard/Memory>
+   Time <standard/Time>
+
+   
+Files/IO
+--------
+.. toctree::
+   :maxdepth: 1
+
+   FileSystem <standard/FileSystem>
+   IO <standard/IO>
+   Path <standard/Path>
+   Regexp <standard/Regexp>
+
+
+System/Interoperability
+-----------------------
+
+.. toctree::
+   :maxdepth: 1
+
+   CPtr <standard/CPtr>
+   DateTime <standard/DateTime>
+   Spawn <standard/Spawn>
+   Sys <standard/Sys>
+   SysBasic <standard/SysBasic>
+   SysCTypes <standard/SysCTypes>
+   SysError <standard/SysError>
+
+Language
```

Comment:
This may be another one that I'd do differently. Maybe rename it to "Utilities", and move "Types" under "System/Interop". (If this was named "Utilities", I think it could be a good home for `Regexp`, too)

Reply:
I like Engin's idea of calling this section "Utilities". It is slightly more descriptive than "Language".

Then, I'd add these modules to this section: Time, DateTime, Builtins, Regexp.

I disagree that Types should go under Interop. At least most of it is purely Chapel.

Reply:
I renamed Language to Language Support, but moved most things that had been under it into utilities.  To me reflection (and to a decent extent, Types) seem more directly related to the language than the other utilities (though Help is on the cusp for me).